CODEOWNERS

```diff
@@ -20,3 +20,5 @@
 /storage/mysql/schema/*          @mhutchinson @AlCutter
 /storage/cloudspanner/spanner.sdl @mhutchinson @AlCutter
 /storage/postgresql/schema/*     @mhutchinson @AlCutter
+
+/util/election2/k8s/*            @osmman
```

Comment:
Unfortunately this doesn't work as only the core team has write access to the repo :-(

Please revert this change.

Reply:
No worries, very easy mistake to make unfortunately!